dubbo-registry/dubbo-registry-api/src/main/java/org/apache/dubbo/registry/integration/RegistryProtocol.java

```diff
@@ -171,6 +171,9 @@ public void unregister(URL registryUrl, URL registeredProviderUrl) {
         // url to export locally
         URL providerUrl = getProviderUrl(originInvoker);
 
+        //export invoker
+        final ExporterChangeableWrapper<T> exporter = doLocalExport(originInvoker, providerUrl);

```

Comment:
Hi, this change looks break the logic, I got the following exception during start. I use Dubbo 2.7 + Nacos. The service is not register to Nacos.

```
java.lang.NullPointerException
	at org.apache.dubbo.registry.integration.RegistryProtocol$OverrideListener.doOverrideIfNecessary(RegistryProtocol.java:561)
	at org.apache.dubbo.registry.integration.RegistryProtocol$ServiceConfigurationListener.notifyOverrides(RegistryProtocol.java:604)
	at org.apache.dubbo.registry.integration.AbstractConfiguratorListener.process(AbstractConfiguratorListener.java:72)
	at org.apache.dubbo.configcenter.support.nacos.NacosDynamicConfiguration$NacosConfigListener.lambda$innerReceive$0(NacosDynamicConfiguration.java:244)
	at java.base/java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:803)
	at java.base/java.util.concurrent.CopyOnWriteArraySet.forEach(CopyOnWriteArraySet.java:425)
	at org.apache.dubbo.configcenter.support.nacos.NacosDynamicConfiguration$NacosConfigListener.innerReceive(NacosDynamicConfiguration.java:244)
	at com.alibaba.nacos.api.config.listener.AbstractSharedListener.receiveConfigInfo(AbstractSharedListener.java:37)
	at com.alibaba.nacos.client.config.impl.CacheData$1.run(CacheData.java:188)
	at com.alibaba.nacos.client.config.impl.CacheData.safeNotifyListener(CacheData.java:209)
	at com.alibaba.nacos.client.config.impl.CacheData.checkListenerMd5(CacheData.java:160)
	at com.alibaba.nacos.client.config.impl.ClientWorker$LongPollingRunnable.run(ClientWorker.java:505)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
```